templates/base.html

```diff
@@ -9,32 +9,33 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
       integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
       crossorigin="anonymous">
-  <!-- Add additional CSS in static file -->
+  
   {% load static %}
   <link rel="stylesheet" href="{% static 'css/styles.css' %}">
 </head>
 
 <body>
-<div class="container-fluid">
-  <div class="row">
-    <div class="col-sm-2">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-sm-2">
 
       {% block sidebar %}
         {% include "includes/sidebar.html" %}
       {% endblock %}
 
-    </div>
-    <div class="col-sm-10 ">
+      </div>
+      <div class="col-sm-10 ">
 
       {% block content %}{% endblock %}
 
       {% block pagination %}
         {% include "includes/pagination.html" %}
+        <p>You have visited this page {{ num_visits }} time{{ num_visits|pluralize }}</p>
```

Comment:
The `pluralize` filter should be applied directly to the variable without spaces. Change `{{ num_visits|pluralize }}` to `{{ num_visits|pluralize }}` to ensure correct pluralization.